app/assets/javascripts/plate-map/plate-layout.js

```diff
@@ -167,5 +173,112 @@ $.widget("DNA.plateLayOut", {
   // wellsData follows syntax: {0:{field1: val1, field2: val2}, 1:{field1: val1, field2: val2}}
   getWellsDifferences: function(wellsData) {
     return this.getDifferentWellsVals(wellsData);
+  },
+
+  disableAllFields: function(){
+    var numericFields = $('.plate-setup-tab-input, .plate-setup-tab-label-select-field, ' +
```

Comment:
Preferably I would like to see field disabling handled by iterating over fields and calling `field.setEnabled(false);`, with the field disabling its components as it sees fit. 

At minimum, this should be doing `var numericFields = this.tabDataContainer.find('.plate-setup-tab-input, ...');